client/web/web.go

```diff
@@ -490,21 +490,35 @@ func (s *Server) serveAPI(w http.ResponseWriter, r *http.Request) {
 }
 
 type nodeData struct {
```

Comment:
Didn't change the names of any fields. Purely additive with moving some of the fields around to a more-sensical grouping. Ideally we would add `json:` specs to each of them, and have them camelcase in the json conversion, but then we'll have to maintain two different structs until the legacy client goes away. Opted to not touch that at all in this change, we can do that in a follow-up if desired.

Reply:
agreed that I wouldn't worry too much  about this, given that legacy will be going away soon

Reply:
yeah i guess we could always migrate it to camel case on the next release after legacy goes away (or the same one)

Reply:
will we even still have this endpoint after legacy goes away, or does it get replaced with a proxied localapi call?  🤷🏻

Reply:
I suppose we'll still need it, since there is additional data we're adding here that doesn't exist in localapi

Reply:
oh yeah good point

Reply:
hahah localapi is not `json:` specced either 😅 so maybe better to just use PascalCase here for consistency